dpc-sdp/sdp-platform/assets/.docker/Dockerfile.cli

```diff
@@ -0,0 +1,33 @@
+# @see https://github.com/dpc-sdp/bay/blob/master/bay/images/Dockerfile.php
+FROM singledigital/bay-cli:latest
+ARG COMPOSER
+
+ENV WEBROOT=docroot \
+    COMPOSER_ALLOW_SUPERUSER=1 \
+    COMPOSER_CACHE_DIR=/tmp/.composer/cache \
+    MYSQL_HOST=mariadb \
+    COMPOSER=${COMPOSER:-composer.json} \
+    DRUPAL_MODULE_PREFIX=%%DRUPAL_MODULE_PREFIX%%
```

Comment:
The string `%%DRUPAL_MODULE_PREFIX%%` will be replaced during install/update. Ultimately this should be from something like `.sdp-config.yml` or similar but is hard wired now just to test it works.